src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,184 @@
+package duke;
+
+import duke.exceptions.*;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.Scanner;
+
+public class Duke {
+    private static final String COMMAND_EXIT_WORD = "bye";
+    private static final String COMMAND_LIST_WORD = "list";
+    private static final String COMMAND_DONE_WORD = "done";
+    private static final String COMMAND_TODO_WORD = "todo";
+    private static final String COMMAND_DEADLINE_WORD = "deadline";
+    private static final String COMMAND_EVENT_WORD = "event";
```

Comment:
Would it be better if you name your constants with the command words grouped at the front? For example, `COMMAND_WORD_EXIT`.